ext/opentelemetry-ext-datadog/tests/test_datadog_exporter.py

```diff
@@ -222,8 +222,8 @@ def test_export(self):
     def test_resources(self):
         test_attributes = [
             {},
-            {"http.method": "GET", "http.route": "/foo"},
-            {"http.method": "GET", "http.path": "/foo"},
+            {"http.method": "GET", "http.route": "/foo/<int:id>"},
```

Comment:
might be good to add a test case with both http.route and http.target (a common scenario) and ensure that the precedence is correct?

Reply:
I went with the conservative approach you suggested, not backing off to `http.target`, being that it would create a cardinality issue.